ambry-server/src/main/java/com.github.ambry.server/ErrorMapping.java

```diff
@@ -35,7 +35,9 @@
     tempMap.put(StoreErrorCodes.ID_Not_Found, ServerErrorCode.Blob_Not_Found);
     tempMap.put(StoreErrorCodes.TTL_Expired, ServerErrorCode.Blob_Expired);
     tempMap.put(StoreErrorCodes.Already_Exist, ServerErrorCode.Blob_Already_Exists);
-    tempMap.put(StoreErrorCodes.Authorization_Failure, ServerErrorCode.Blob_Not_Found);
+    tempMap.put(StoreErrorCodes.Authorization_Failure, ServerErrorCode.Blob_Authorization_Failure);
```

Comment:
I suggest not to make this change in this PR. 
Router doesn't have the code to handle `ServerErrorCode.Blob_Authorization_Failure` in this PR. Level it to #867 .

Reply:
Done